tests/DsnTest.php

```diff
@@ -171,6 +171,16 @@ public function createFromStringThrowsExceptionIfValueIsInvalidDataProvider(): \
             'tcp://public:secret@example.com/1',
             'The scheme of the "tcp://public:secret@example.com/1" DSN must be either "http" or "https".',
         ];
+
+        yield 'valid project id' => [
+            'http://public:secret@example.com/j',
+            'DSN must contain a valid project ID.',
+        ];
+
+        yield 'valid project id' => [
```

Comment:
Please change this description to `invalid project ID (negative number)` to make it different from the one before